### DISCUSSION TITLE
Profile Update Error

### DISCUSSION THREAD

**Discussion by kiranharidas187** [Sunbird-Ed/Community#6] [Installation] [answered]
After setting up Shikshalokam instance of Sunbird and trying to update the board, medium, subject, etc we are facing an error. Request details are as follows

Request
`curl 'https://dhruva.shikshalokam.org/learner/user/v3/update' \
  -X 'PATCH' \
  -H 'Accept: application/json' \
  -H 'Accept-Language: en-US,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Origin: https://dhruva.shikshalokam.org' \
  -H 'Referer: https://dhruva.shikshalokam.org/profile' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36' \
  -H 'X-App-Id: slecosystem.slecosystem.portal' \
  -H 'X-App-Version: 4.6.0' \
  -H 'X-Channel-Id: 0124487522476933120' \
  -H 'X-Device-ID: 6b186526688e67853bdadd311e3d7f29' \
  -H 'X-Org-code: 0124487522476933120' \
  -H 'X-Request-ID: 06dccb9d-f008-2f4c-d88a-44f425203dbc' \
  -H 'X-Session-ID: Puu3D4nJsyM0zQam3RAqetkqRmCWwpdp' \
  -H 'X-Source: web' \
  -H 'X-User-ID: 0edbd37c-5613-47c5-90cd-f21898a08b25' \
  -H 'X-msgid: 06dccb9d-f008-2f4c-d88a-44f425203dbc' \
  -H 'sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="101", "Google Chrome";v="101"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "macOS"' \
  -H 'ts: 2022-05-11T12:30:24+05:30' \
  --data-raw '{"params":{},"request":{"framework":{"id":"NCF","board":["NCF framework"],"medium":["English"],"gradeLevel":["Class 1"],"subject":["Accountancy and Auditing"]},"userId":"0edbd37c-5613-47c5-90cd-f21898a08b25"}}' \
  --compressed`

Response
`{"id":"api.user.update","ver":"v3","ts":"2022-05-11 07:08:10:206+0000","params":{"resmsgid":null,"msgid":"06583b13-fd0b-3d86-d7be-5e9de61c1461","err":"INVALID_PARAMETER_VALUE","status":"INVALID_PARAMETER_VALUE","errmsg":"Invalid value NCF framework for parameter framework.board. Please provide a valid value."},"responseCode":"CLIENT_ERROR","result":{}}`

**Note**
We have just one framework called NCF.  Board category and terms under the board are already available.

Please suggest what could be the solution
@sharathkashyap  @Krishnaj20
cc - @vijiurs @aks30

**Comment by Krishnaj20** [comment, marked as answer]
@kiranharidas187 - pls tag @luckynvdu for all Ed related issues/discussions.

**Reply by sharathkashyap** [reply]
@kiranharidas187 Can you kindly use the code formatter for curl and provide us with code formatted text

**Reply by swayangjit** [reply]
@kiranharidas187  The specific framework might not be available or the BMGS value might not be available in the framework. Can you please get a framework call and check if everything is available for that specific framework?

**Reply by kiranharidas187** [reply]
`{
  "id": "api.framework.read",
  "ver": "1.0",
  "ts": "2022-05-11T10:18:03.468Z",
  "params": {
    "resmsgid": "a46aa8c0-d113-11ec-9bab-9dca33b46a0e",
    "msgid": "48dc3ea6-aeb7-b826-fca5-24b59b0e597c",
    "status": "successful",
    "err": null,
    "errmsg": null
  },
  "responseCode": "OK",
  "result": {
    "framework": {
      "identifier": "NCF",
      "code": "NCF",
      "name": "NCF framework",
      "description": "NCF",
      "categories": [
        {
          "identifier": "ncf_board",
          "code": "board",
          "terms": [
            {
              "identifier": "ncf_board_ncert",
              "code": "ncert",
              "translations": null,
              "name": "NCERT",
              "description": "",
              "index": 1,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_cbse",
              "code": "cbse",
              "translations": null,
              "name": "CBSE",
              "description": "",
              "index": 2,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_icse",
              "code": "icse",
              "translations": null,
              "name": "ICSE",
              "description": "",
              "index": 3,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_upboard",
              "code": "upboard",
              "translations": null,
              "name": "State (Uttar Pradesh)",
              "description": "State (Uttar Pradesh)",
              "index": 4,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_apboard",
              "code": "apboard",
              "translations": null,
              "name": "State (Andhra Pradesh)",
              "description": "State (Andhra Pradesh)",
              "index": 5,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_tnboard",
              "code": "tnboard",
              "translations": null,
              "name": "State (Tamil Nadu)",
              "description": "State (Tamil Nadu)",
              "index": 6,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_mscert",
              "code": "mscert",
              "translations": null,
              "name": "State (Maharashtra)",
              "description": "State (Maharashtra)",
              "index": 8,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_bser",
              "code": "bser",
              "translations": null,
              "name": "State (Rajasthan)",
              "description": "State (Rajasthan)",
              "index": 9,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_others",
              "code": "others",
              "translations": null,
              "name": "Other",
              "description": "Other",
              "index": 10,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_dscertka",
              "code": "dscertka",
              "translations": null,
              "name": "State (Karnataka)",
              "description": "State (Karnataka)",
              "index": 11,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_stateassam",
              "code": "stateassam",
              "translations": null,
              "name": "State (Assam)",
              "description": "State (Assam)",
              "index": 13,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_statejammukashmir",
              "code": "statejammu&kashmir",
              "translations": null,
              "name": "State(Jammu & Kashmir)",
              "description": "State(Jammu & Kashmir)",
              "index": 14,
              "category": "board",
              "status": "Live"
            },
            {
              "identifier": "ncf_board_ncf",
              "code": "NCF",
              "translations": null,
              "name": "NCF framework",
              "description": "NCF framework",
              "index": 15,
              "category": "board",
              "status": "Live"
            }
          ],
          "translations": null,
          "name": "Board",
          "description": "",
          "index": 1,
          "status": "Live"
        },
        {
          "identifier": "ncf_medium",
          "code": "medium",
          "terms": [
            {
              "identifier": "ncf_medium_english",
              "code": "english",
              "translations": null,
              "name": "English",
              "description": "",
              "index": 1,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_hindi",
              "code": "hindi",
              "translations": null,
              "name": "Hindi",
              "description": "",
              "index": 2,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_oriya",
              "code": "oriya",
              "translations": null,
              "name": "Odia",
              "description": "Odia",
              "index": 3,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_telugu",
              "code": "telugu",
              "translations": null,
              "name": "Telugu",
              "description": "",
              "index": 4,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_kannada",
              "code": "kannada",
              "translations": null,
              "name": "Kannada",
              "description": "",
              "index": 5,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_marathi",
              "code": "marathi",
              "translations": null,
              "name": "Marathi",
              "description": "",
              "index": 6,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_assamese",
              "code": "assamese",
              "translations": null,
              "name": "Assamese",
              "description": "",
              "index": 7,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_bengali",
              "code": "bengali",
              "translations": null,
              "name": "Bengali",
              "description": "",
              "index": 8,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_gujarati",
              "code": "gujarati",
              "translations": null,
              "name": "Gujarati",
              "description": "",
              "index": 9,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_tamil",
              "code": "tamil",
              "translations": null,
              "name": "Tamil",
              "description": "",
              "index": 10,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_urdu",
              "code": "urdu",
              "translations": null,
              "name": "Urdu",
              "description": "",
              "index": 11,
              "category": "medium",
              "status": "Live"
            },
            {
              "identifier": "ncf_medium_other",
              "code": "other",
              "translations": null,
              "name": "Other",
              "description": "",
              "index": 12,
              "category": "medium",
              "status": "Live"
            }
          ],
          "translations": null,
          "name": "Medium",
          "description": "",
          "index": 2,
          "status": "Live"
        },
        {
          "identifier": "ncf_gradelevel",
          "code": "gradeLevel",
          "terms": [
            {
              "identifier": "ncf_gradelevel_kindergarten",
              "code": "kindergarten",
              "translations": null,
              "name": "KG",
              "description": "KG",
              "index": 1,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade1",
              "code": "grade1",
              "translations": null,
              "name": "Class 1",
              "description": "Class 1",
              "index": 2,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade2",
              "code": "grade2",
              "translations": null,
              "name": "Class 2",
              "description": "Class 2",
              "index": 3,
              "category": "gradeLevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade3",
              "code": "grade3",
              "translations": null,
              "name": "Class 3",
              "description": "Class 3",
              "index": 4,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade4",
              "code": "grade4",
              "translations": null,
              "name": "Class 4",
              "description": "Class 4",
              "index": 5,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade5",
              "code": "grade5",
              "translations": null,
              "name": "Class 5",
              "description": "Class 5",
              "index": 6,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade6",
              "code": "grade6",
              "translations": null,
              "name": "Class 6",
              "description": "Class 6",
              "index": 7,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade7",
              "code": "grade7",
              "translations": null,
              "name": "Class 7",
              "description": "Class 7",
              "index": 8,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade8",
              "code": "grade8",
              "translations": null,
              "name": "Class 8",
              "description": "Class 8",
              "index": 9,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade9",
              "code": "grade9",
              "translations": null,
              "name": "Class 9",
              "description": "Class 9",
              "index": 10,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade10",
              "code": "grade10",
              "translations": null,
              "name": "Class 10",
              "description": "Class 10",
              "index": 11,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade11",
              "code": "grade11",
              "translations": null,
              "name": "Class 11",
              "description": "Class 11",
              "index": 12,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_grade12",
              "code": "grade12",
              "translations": null,
              "name": "Class 12",
              "description": "Class 12",
              "index": 13,
              "category": "gradelevel",
              "status": "Live"
            },
            {
              "identifier": "ncf_gradelevel_others",
              "code": "others",
              "translations": null,
              "name": "Other",
              "description": "",
              "index": 14,
              "category": "gradeLevel",
              "status": "Live"
            }
          ],
          "translations": null,
          "name": "Class",
          "description": "",
          "index": 3,
          "status": "Live"
        },
        {
          "identifier": "ncf_subject",
          "code": "subject",
          "terms": [
            {
              "identifier": "ncf_subject_mathematics",
              "code": "mathematics",
              "translations": null,
              "name": "Mathematics",
              "description": "",
              "index": 1,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_english",
              "code": "english",
              "translations": null,
              "name": "English",
              "description": "",
              "index": 2,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_tamil",
              "code": "tamil",
              "translations": null,
              "name": "Tamil",
              "description": "",
              "index": 3,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_telugu",
              "code": "telugu",
              "translations": null,
              "name": "Telugu",
              "description": "",
              "index": 4,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_geography",
              "code": "geography",
              "translations": null,
              "name": "Geography",
              "description": "",
              "index": 5,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_urdu",
              "code": "urdu",
              "translations": null,
              "name": "Urdu",
              "description": "",
              "index": 6,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_kannada",
              "code": "kannada",
              "translations": null,
              "name": "Kannada",
              "description": "",
              "index": 7,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_assamese",
              "code": "assamese",
              "translations": null,
              "name": "Assamese",
              "description": "",
              "index": 8,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_physics",
              "code": "physics",
              "translations": null,
              "name": "Physics",
              "description": "",
              "index": 9,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_chemistry",
              "code": "chemistry",
              "translations": null,
              "name": "Chemistry",
              "description": "",
              "index": 10,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_hindi",
              "code": "hindi",
              "translations": null,
              "name": "Hindi",
              "description": "",
              "index": 11,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_marathi",
              "code": "marathi",
              "translations": null,
              "name": "Marathi",
              "description": "",
              "index": 12,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_environmentalstudies",
              "code": "environmentalstudies",
              "translations": null,
              "name": "EvS",
              "description": "EvS",
              "index": 13,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_politicalscience",
              "code": "politicalscience",
              "translations": null,
              "name": "Political Science",
              "description": "",
              "index": 14,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_bengali",
              "code": "bengali",
              "translations": null,
              "name": "Bengali",
              "description": "",
              "index": 15,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_history",
              "code": "history",
              "translations": null,
              "name": "History",
              "description": "",
              "index": 16,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_gujarati",
              "code": "gujarati",
              "translations": null,
              "name": "Gujarati",
              "description": "",
              "index": 17,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_biology",
              "code": "biology",
              "translations": null,
              "name": "Biology",
              "description": "",
              "index": 18,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_oriya",
              "code": "oriya",
              "translations": null,
              "name": "Odia",
              "description": "Odia",
              "index": 19,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_punjabi",
              "code": "punjabi",
              "translations": null,
              "name": "Punjabi",
              "description": "",
              "index": 20,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_nepali",
              "code": "nepali",
              "translations": null,
              "name": "Nepali",
              "description": "",
              "index": 21,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_malayalam",
              "code": "malayalam",
              "translations": null,
              "name": "Malayalam",
              "description": "",
              "index": 22,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_sanskrit",
              "code": "sanskrit",
              "translations": null,
              "name": "Sanskrit",
              "description": "Sanskrit",
              "index": 23,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_socialstudies",
              "code": "socialstudies",
              "translations": null,
              "name": "Social Studies",
              "description": "Social Studies",
              "index": 24,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_science",
              "code": "science",
              "translations": null,
              "name": "Science",
              "description": "Science",
              "index": 25,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_healthandphysicaleducation",
              "code": "healthandphysicaleducation",
              "translations": null,
              "name": "Health and Physical Education",
              "description": "Health and Physical Education",
              "index": 26,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_economics",
              "code": "economics",
              "translations": null,
              "name": "Economics",
              "description": "Economics",
              "index": 27,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_other",
              "code": "other",
              "translations": null,
              "name": "Other",
              "description": "Other",
              "index": 28,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_botany",
              "code": "botany",
              "translations": null,
              "name": "Botany",
              "description": "Botany",
              "index": 29,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_zoology",
              "code": "zoology",
              "translations": null,
              "name": "Zoology",
              "description": "Zoology",
              "index": 30,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_commerce",
              "code": "commerce",
              "translations": null,
              "name": "Commerce",
              "description": "Commerce",
              "index": 31,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_accountancy",
              "code": "accountancy",
              "translations": null,
              "name": "Accountancy",
              "description": "Accountancy",
              "index": 32,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_statistics",
              "code": "statistics",
              "translations": null,
              "name": "Statistics",
              "description": "Statistics",
              "index": 33,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_computerscience",
              "code": "computerscience",
              "translations": null,
              "name": "Computer Science",
              "description": "Computer Science",
              "index": 34,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_computerapplications",
              "code": "computerapplications",
              "translations": null,
              "name": "Computer Applications",
              "description": "Computer Applications",
              "index": 35,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_biochemistry",
              "code": "biochemistry",
              "translations": null,
              "name": "Bio Chemistry",
              "description": "Bio Chemistry",
              "index": 36,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_microbiology",
              "code": "microbiology",
              "translations": null,
              "name": "Micro Biology",
              "description": "Micro Biology",
              "index": 37,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_homescience",
              "code": "homescience",
              "translations": null,
              "name": "Home Science",
              "description": "Home Science",
              "index": 38,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_nutritiondiabetics",
              "code": "nutritiondiabetics",
              "translations": null,
              "name": "Nutrition Diabetics",
              "description": "Nutrition Diabetics",
              "index": 39,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_communicativeenglish",
              "code": "communicativeenglish",
              "translations": null,
              "name": "Communicative English",
              "description": "Communicative English",
              "index": 40,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_advancetamil",
              "code": "advancetamil",
              "translations": null,
              "name": "Advance Tamil",
              "description": "Advance Tamil",
              "index": 41,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_businessmaths",
              "code": "businessmaths",
              "translations": null,
              "name": "Business Maths",
              "description": "Business Maths",
              "index": 42,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_ethicsindianculture",
              "code": "ethicsindianculture",
              "translations": null,
              "name": "Ethics Indian Culture",
              "description": "Ethics Indian Culture",
              "index": 43,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_computertechnology",
              "code": "computertechnology",
              "translations": null,
              "name": "Computer Technology",
              "description": "Computer Technology",
              "index": 44,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_foodmanagement",
              "code": "foodmanagement",
              "translations": null,
              "name": "Food Management",
              "description": "Food Management",
              "index": 45,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_officemanagement",
              "code": "officemanagement",
              "translations": null,
              "name": "Office Management",
              "description": "Office Management",
              "index": 46,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_accountancyauditing",
              "code": "accountancyauditing",
              "translations": null,
              "name": "Accountancy Auditing",
              "description": "Accountancy Auditing",
              "index": 47,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_agriculture",
              "code": "agriculture",
              "translations": null,
              "name": "Agriculture",
              "description": "Agriculture",
              "index": 48,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_generalmachinist",
              "code": "generalmachinist",
              "translations": null,
              "name": "General Machinist",
              "description": "General Machinist",
              "index": 49,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_electricalmachinesappliances",
              "code": "electricalmachinesappliances",
              "translations": null,
              "name": "Electrical Machines Appliances",
              "description": "Electrical Machines Appliances",
              "index": 50,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_electricalequipments",
              "code": "electricalequipments",
              "translations": null,
              "name": "Electrical Equipments",
              "description": "Electrical Equipments",
              "index": 51,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_draughtsmancivil",
              "code": "draughtsmancivil",
              "translations": null,
              "name": "Draughtsman Civil",
              "description": "Draughtsman Civil",
              "index": 52,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_automechanic",
              "code": "automechanic",
              "translations": null,
              "name": "Auto Mechanic",
              "description": "Auto Mechanic",
              "index": 53,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_textiletechnology",
              "code": "textiletechnology",
              "translations": null,
              "name": "Textile Technology",
              "description": "Textile Technology",
              "index": 54,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_textiledressdesigning",
              "code": "textiledressdesigning",
              "translations": null,
              "name": "Textile Dress Designing",
              "description": "Textile Dress Designing",
              "index": 55,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_nutritiondietics",
              "code": "nutritiondietics",
              "translations": null,
              "name": "Nutrition Dietics",
              "description": "Nutrition Dietics",
              "index": 56,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_ethicsandindianculture",
              "code": "ethicsandindianculture",
              "translations": null,
              "name": "Ethics and Indian Culture",
              "description": "Ethics and Indian Culture",
              "index": 57,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_foodservicemanagement",
              "code": "foodservicemanagement",
              "translations": null,
              "name": "Food Service Management",
              "description": "Food Service Management",
              "index": 58,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_officemanagementandsecretaryship",
              "code": "officemanagementandsecretaryship",
              "translations": null,
              "name": "Office Management and Secretaryship",
              "description": "Office Management and Secretaryship",
              "index": 59,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_accountancyandauditing",
              "code": "accountancyandauditing",
              "translations": null,
              "name": "Accountancy and Auditing",
              "description": "Accountancy and Auditing",
              "index": 60,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_agriculturalscience",
              "code": "agriculturalscience",
              "translations": null,
              "name": "Agricultural Science",
              "description": "Agricultural Science",
              "index": 61,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_basicmechanicalengineering",
              "code": "basicmechanicalengineering",
              "translations": null,
              "name": "Basic Mechanical Engineering",
              "description": "Basic Mechanical Engineering",
              "index": 62,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_basicelectricalengineering",
              "code": "basicelectricalengineering",
              "translations": null,
              "name": "Basic Electrical Engineering",
              "description": "Basic Electrical Engineering",
              "index": 63,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_basicelectronicsengineering",
              "code": "basicelectronicsengineering",
              "translations": null,
              "name": "Basic Electronics Engineering",
              "description": "Basic Electronics Engineering",
              "index": 64,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_basiccivilengineering",
              "code": "basiccivilengineering",
              "translations": null,
              "name": "Basic Civil Engineering",
              "description": "Basic Civil Engineering",
              "index": 65,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_basicautomobileengineering",
              "code": "basicautomobileengineering",
              "translations": null,
              "name": "Basic Automobile Engineering",
              "description": "Basic Automobile Engineering",
              "index": 66,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_textilesdressdesigning",
              "code": "textilesdressdesigning",
              "translations": null,
              "name": "Textiles &  Dress Designing",
              "description": "Textiles &  Dress Designing",
              "index": 67,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_nursingcore",
              "code": "nursingcore",
              "translations": null,
              "name": "Nursing Core",
              "description": "Nursing Core",
              "index": 68,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_tourismandtravelmanagement",
              "code": "tourismandtravelmanagement",
              "translations": null,
              "name": "Tourism and Travel Management",
              "description": "Tourism and Travel Management",
              "index": 69,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_horticulture",
              "code": "horticulture",
              "translations": null,
              "name": "Horticulture",
              "description": "Horticulture",
              "index": 70,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_healthbeautystudies",
              "code": "healthbeautystudies",
              "translations": null,
              "name": "Health & Beauty Studies",
              "description": "Health & Beauty Studies",
              "index": 71,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_secretarialpractices",
              "code": "secretarialpractices",
              "translations": null,
              "name": "Secretarial Practices",
              "description": "Secretarial Practices",
              "index": 72,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_costaccountingtaxation",
              "code": "costaccountingtaxation",
              "translations": null,
              "name": "Cost Accounting & Taxation",
              "description": "Cost Accounting & Taxation",
              "index": 73,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_animalsciencetechnology",
              "code": "animalsciencetechnology",
              "translations": null,
              "name": "Animal Science & Technology",
              "description": "Animal Science & Technology",
              "index": 74,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_chemicaltechnology",
              "code": "chemicaltechnology",
              "translations": null,
              "name": "Chemical Technology",
              "description": "Chemical Technology",
              "index": 75,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_plastictechnology",
              "code": "plastictechnology",
              "translations": null,
              "name": "Plastic Technology",
              "description": "Plastic Technology",
              "index": 76,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_fisheriesandaquaculture",
              "code": "fisheriesandaquaculture",
              "translations": null,
              "name": "Fisheries and Aquaculture",
              "description": "Fisheries and Aquaculture",
              "index": 77,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_artandarchitecture",
              "code": "artandarchitecture",
              "translations": null,
              "name": "Art and Architecture",
              "description": "Art and Architecture",
              "index": 78,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_hospitalmanagement",
              "code": "hospitalmanagement",
              "translations": null,
              "name": "Hospital Management",
              "description": "Hospital Management",
              "index": 79,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_fashiontechnology",
              "code": "fashiontechnology",
              "translations": null,
              "name": "Fashion Technology",
              "description": "Fashion Technology",
              "index": 80,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_nursingvocational",
              "code": "nursingvocational",
              "translations": null,
              "name": "Nursing Vocational",
              "description": "Nursing Vocational",
              "index": 81,
              "category": "subject",
              "status": "Live"
            },
            {
              "identifier": "ncf_subject_coreandvocational",
              "code": "coreandvocational",
              "translations": null,
              "name": "Core and Vocational",
              "description": "Core and Vocational",
              "index": 82,
              "category": "subject",
              "status": "Live"
            }
          ],
          "translations": null,
          "name": "Subject",
          "description": "",
          "index": 4,
          "status": "Live"
        }
      ],
      "type": "K-12",
      "objectType": "Framework"
    }
  }
}`

@swayangjit Its there

**Reply by swayangjit** [reply]
@kiranharidas187 As discussed please redeploy the framework service and check

**Reply by kiranharidas187** [reply]
@swayangjit After restarting the service profile updating started working. So the takeaway is after the framework update we will have to restart the service.

Thank you team for the support
